cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/bulkwriter/TableSchemaTest.java

```diff
@@ -231,4 +234,10 @@ private TableSchemaTestCommon.MockTableSchemaBuilder getValidSchemaBuilder()
                 .withWriteMode(WriteMode.INSERT)
                 .withDataFrameSchema(validDataFrameSchema);
     }
+
+    // Removes the unique table name to make validation consistent
+    private static String replaceTableName(String statement)
```

Comment:
nit: how about rename to `trimTestTableId`?

Reply:
table Id might cause confusion with Cassandra's table ID. I will rename it to `trimUniqueTableName`